components/ContentContainer.tsx

```diff
@@ -0,0 +1,9 @@
+import { Container } from "@mantine/core";
+
+export interface ContentContainerProps {
+  children: React.ReactNode;
+}
+
+export const ContentContainer: React.FC<ContentContainerProps> = ({ children }) => (
+  <Container size={1250}>{children}</Container>
```

Comment:
I guess we can't override the matine "lg" right?

Reply:
I think we can do once we've updated to the newer version of mantine. Will try that soon and see if it causes any issues. But in the meantime i think we can commit this